lib/src/main/java/org/exoplatform/social/addons/search/ProfileSearchConnector.java

```diff
@@ -254,10 +254,10 @@ private String buildTypeEx(Type type) {
         result = "connections";
         break;
       case INCOMING:
-        result = "incomings";
+        result = "outgoings";
         break;
       case OUTGOING:
-        result = "outgoings";
+        result = "incomings";
```

Comment:
Actually the field "outgoing" contains all IDs of Identities who receive the connection request from current user. When userA sends a request connect to userB then userA will be contained on "incoming" field of userB and userB will be contained on "outgoing" field of userA.

So, If we want to search on "outgoing" of a people, we have to search on all Identities who contain his ID on their "incoming" field. That's why I reverse the "incoming" and "outgoing".